src/vs/workbench/contrib/accessibility/browser/accessibilityContribution.ts

```diff
@@ -65,6 +69,12 @@ const configuration: IConfigurationNode = {
 		[AccessibilityVerbositySettingId.Notification]: {
 			description: localize('verbosity.notification', 'Provide information about how to open the notification in an accessible view.'),
 			...baseProperty
+		},
+		[AccessibilitySettingId.UnfocusedViewOpacity]: {
+			description: localize('unfocusedViewOpacity', 'The opacity percentage (0.0 to 1.0) to use for unfocused editors and terminals.'),
+			type: 'number',
+			default: 1,
+			tags: ['accessibility']
 		}
```

Comment:
@meganrogge this registration needs to be shared between all `accessibiity.*` settings, so we should move it somewhere more central.

Reply:
As in moving this file to `platform`?

Reply:
Yeah that's been confusing to me and annoying because I always open the wrong one 😆 . Maybe i should consolidate them?

Reply:
https://github.com/microsoft/vscode/issues/189318